backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/services/EditService.java

```diff
@@ -86,6 +86,9 @@ public EditService(IEditingContextEPackageService editingContextEPackageService,
     }
 
     private Optional<EClass> getEClass(EPackage.Registry ePackageRegistry, String kind) {
+        if (kind == null || kind.isBlank()) {
+            return Optional.empty();
+        }
```

Comment:
This is just a workaround for the current issues with the changes on `kind` syntax. Not sure yet what would be a proper fix, but without at least this workaround I can not open a diagram, so can not test anything.

Reply:
The proper fix needs to be done in Sirius Web and the whole method should be removed from Sirius Components since it is unused otherwise.

Reply:
Working on this (in a separate PR).